tasks/test.rake

```diff
@@ -4,17 +4,21 @@ require 'rspec/core/rake_task'
 ## spec / unit tests
 ##
 
+vendor_selector = lambda do |spec_file|
+  spec_file.start_with?('vendor/')
```

Comment:
Is this a temporary change that goes along with the travis change?

Reply:
This is not temporary; it is for Travis cache.

To elaborate on it, when you use
~~~yml
cache: bundler
~~~
on Travis it would install gems inside `vendor/` instead of global system location, and the caching module would pick up files from there.

Since this repository use
~~~Ruby
Dir.glob('**/spec')
~~~
to search for testing, it would pick up 3rd party spec files as well, since the files are now located under `./vendor`.

This filter has been written to allow spec files to exist under `./vendor` without interfering with the actual tests intended.

Reply:
Ah, I see. We should definitely not be running those tests. I would assume they only exist in gems/* and build_tools.